src/userList2.js

```diff
@@ -0,0 +1,26 @@
+import React from "react"
+import UserList2ClientBreakdown from "./userList2ClientBreakdown";
+
+export default function UserList2(props) {
```

Comment:
arrow function notation with destructuring
```suggestion
const UserList2 = ({ userDataList }) => {
```